s3fs/core.py

```diff
@@ -1040,6 +1040,10 @@ def walk(self, path, maxdepth=None, **kwargs):
             raise ValueError('Cannot crawl all of S3')
         return super().walk(path, maxdepth=maxdepth, **kwargs)
 
+    def modified(self, path, version_id=None, refresh=False):
+        info = self.info(path=path, version_id=version_id, refresh=refresh)
+        return info['LastModified'].replace(tzinfo=None)
```

Comment:
Buckets have `'CreationDate'` and deeper directories have neither field. I don't know if you intend this to work on directories, but they `KeyError` may come as a surprise.

Reply:
My intention was only to support actual files, not directories (or s3s pseudo directories). Let me push an update.